pkgs/development/python-modules/generic/run_setup.py

```diff
@@ -0,0 +1,6 @@
+import setuptools
```

Comment:
@domenkozar What happened to the docs for this? I'm a little disappointed that this got merged in this state.